training/ikon20/part1-getting-started.ipynb

```diff
@@ -0,0 +1,518 @@
+   "source": [
+    "Note that the exercises in the following are fictional and do not represent the actual data reduction workflow."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 1: Use the HTML representation to see what the loaded data contains\n",
+    "\n",
+    "The HTML representation is what Jupyter displays for a scipp object.\n",
+    "- Take some time to explore this view and try to understand all the information (dimensions, dtypes, units, ...).\n",
+    "- Note that sections can be expanded, and values can shown by clicking the icons to the right."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 2: Plot the data\n",
+    "\n",
+    "Scipp objects can by created using the `plot()` method.\n",
+    "Alternatively `sc.plot.plot(obj)` can be used.\n",
+    "Since this is neutron-scattering data, we can also use the \"instrument view\", provided by `sc.neutron.instrument_view(obj)`.\n",
+    "\n",
+    "- Plot the loaded data and familiarize yourself with the controls.\n",
+    "- Create the instrument view and familiarize yourself with the controls."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data.plot()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# pixel_size is optional, but on certain screens/systems the defaults don't work well\n",
+    "sc.neutron.instrument_view(data, pixel_size=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 3: Exporing meta data\n",
+    "\n",
+    "Above we saw that many attributes are scalar variable with `dtype=DataArray`.\n",
+    "The single value in a scalar variable is accessed using the `value` property.\n",
+    "Compare:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data.attrs['proton_charge_by_period']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data.attrs['proton_charge_by_period'].value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Exercises:\n",
+    "1. Find some attributes of `data` with `dtype=DataArray` and plot their `value`.\n",
+    "   Also try `sc.table(attr.value)` to show a table representation.\n",
+    "2. Find and plot a monitor.\n",
+    "3. Try to normalize `data` to monitor 1.\n",
+    "   Why does this fail?\n",
+    "4. Plot all the monitors on the same plot.\n",
+    "   Note that `sc.plot.plot()` can be used with a Python `dict` for this purpose: `sc.plot.plot({'a':something, 'b':else})`.\n",
```

Comment:
Wouldn't that be more confusing, in a different way?

Reply:
maybe it would be better if it was just `sc.plot()`?
maybe i should rename the module to `plotting` and then just have `plot` as the function. It's basically the only thing we import anyway, apart from some special methods (`superplot`, etc.. which are basically unused)